recipes/iiboost/build.sh

```diff
@@ -0,0 +1,19 @@
+
+#!/bin/bash
+
+if [[ "$(uname)" == "Darwin" ]]; then
+    export ENABLE_OPENMP=no
+else
+    export ENABLE_OPENMP=yes
+fi
+
+mkdir build && cd build
+cmake \
+    -DCMAKE_PREFIX_PATH=$PREFIX \
+    -DITK_DIR=$PREFIX/lib/cmake/ITK-4.11/ \
+    -DIIBOOST_USE_OPENMP=$ENABLE_OPENMP \
+	..
+
+make
```

Comment:
I think setting [`pickle.DEFAULT_PROTOCOL`]( https://docs.python.org/3/library/pickle.html#pickle.DEFAULT_PROTOCOL ) to `0` should fix this issue on Python 3 and be a no-op for Python 2 as `0` appears to be the default on Python 2. Might be worth a try.